CONTRIBUTING.es.md

```diff
@@ -0,0 +1,73 @@
+[![English](https://img.shields.io/badge/language-English-blue.svg)](CONTRIBUTING.md)
```

Comment:
Super good job!
But I would put the language change selector under the Heading title 🍻

Reply:
Oki doki, language change selectors moved to where you´ve requested :) Thanks for your comment!